.github/workflows/validate.yml

```diff
@@ -1,5 +1,16 @@
 name: validate
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      [
+        "+([0-9])?(.{+([0-9]),x}).x",
+        "master",
+        "next",
+        "next-major",
+        "beta",
+        "alpha",
```

Comment:
We only use master out of all of those 😛 do we need the rest?

I'm not sure how automated `kcd-scripts` is - if all of this is automatically added then it makes sense, but if it's more manual we can just keep `master` here

Reply:
It will become handy when we need an extra branch if we want to create automated alpha/beta/next/... releases